multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -159,45 +159,24 @@ coreos:
         ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
         ExecStart=/opt/bin/install-kube-system
 
-{{ if .UseCalico }}
-    - name: install-calico-system.service
-      command: start
-      content: |
-        [Unit]
-        Requires=kubelet.service docker.service
-        After=kubelet.service docker.service
-
-        [Service]
-        Type=simple
-        StartLimitInterval=0
-        Restart=on-failure
-        ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
-        ExecStart=/opt/bin/install-calico-system
-{{ end }}
-
 write_files:
+  - path: /opt/bin/kubectl-common
```

Comment:
Should we keep this separate from the `/opt/bin/install-kube-system`?

If yes, why not rename this to `/opt/bin/kubectl` (and give execution permissions)? That way we may use it as an "kubectl wrapper" everywhere. E.g `/opt/bin/kubectl get nodes`, `/opt/bin/kubectl apply -f ...`